scripts/variantstore/beta_docs/gvs-exome-workspace-description.md

```diff
@@ -0,0 +1,144 @@
+### Genomic Variant Store Exomes workflow (beta) for variant discovery
+
+The [Genomic Variant Store (GVS)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/gvs-product-sheet.pdf) was developed as a solution for variant discovery on a large scale. The GVS is powered by BigQuery and creates large joint callsets more reliably with decreased time and cost compared to previous solutions.
+
+This workspace contains a fully reproducible example workflow for Exomes variant discovery using the GVS workflow.
+
+Scroll down for an overview of the workflow, example data, cost estimates, and additional resources.
+
+The materials in this workspace were developed by the Data Sciences Platform at the Broad Institute.
+
+## Workflow Overview
+
+![A diagram depicting the Genomic Variant Store workflow. Sample GVCF files are imported into the core data model. A filtering model is trained using the VETS toolchain, and then applied while the samples are extracted as cohorts in sharded joint VCF files. Each step integrates BigQuery and GATK tools.](https://storage.googleapis.com/terra-featured-workspaces/Genomic-Variant-Store/genomic-variant-store_diagram.png)
+
+### What does it do?
+
+The [GVS workflow](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) is an open-source, cloud-optimized workflow for joint calling at a large scale using the GVS. The workflow takes in single sample GVCF files, loads them into [BigQuery](https://cloud.google.com/bigquery/docs) tables, and combines them into a variant filtering model driven by machine learning. The model is uploaded back into BigQuery and applied to the data. The workflow produces sharded joint VCF files with indices, a manifest file, and metrics.
+
+For workflow documentation, see the [Genomic Variant Store workflow overview](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/beta_docs/gvs-overview.md).
+
+---
+
+**Important Update:**
+
+Starting September 1 2023, variants in GVS are filtered using the GATK Variant Extract-Train-Score toolchain with an isolation-forest model by default. GVS maintains the ability to run VQSR on callsets up to 10,000 genomes to reproduce the same results from past analyses. See the [release notes](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/docs/release_notes/VETS_Release.pdf) for more information.
+
+---
+
+### What data does it require as input?
```

Comment:
I think this section should mention that it uses xyz interval list for exomes by default, and the linked documentation 'run your own samples' can give a bit more detail about where a readme is about that interval list. I'll update the linked documentation when I do the docs update for exomes.